lib/bridge/github/contents.rb

```diff
@@ -23,7 +23,7 @@ def issue_template_content
         content = nil
       end
 
-      return content
+      return content ? content.force_encoding("utf-8") : content
```

Comment:
Is there a reason to use `content ? content.force_encoding("utf-8") : content` over `content.force_encoding("utf-8") if content`?